app/src/theme/index.js

```diff
@@ -108,7 +108,7 @@ const theme = {
     borderRadius: '8px ',
     boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.05)',
     paddingHorizontal: '25px',
-    paddingVertical: '25px',
+    paddingVertical: '21px',
```

Comment:
This will change all instances that use this value. Is this what we intend?

Reply:
@pimato , what do you think?

Reply:
@liaojikunwork my designs had 21px for the create market widget but I agree with @KadenZipfel that we should stay consistent with 25px vertical padding. Sorry for not having consistency in my designs. Will clean up soon.

Reply:
@pimato , will update it.